### PR TITLE
feat: add flags to charmil crud commands

### LINKED ISSUE #178: Add flags to Charmil CRUD generator templates

### PR DESCRIPTION
**Pull request by namit-chandwani**
Closes #178 

### Description
Added flags to the Charmil CRUD commands.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### Checklist

- [x] Documentation added for the feature
- [x] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: pkg/template/crud/create.go [hunk 1]
```diff
@@ -26,6 +29,10 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
+	// Adds local flags
+	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", "json", f.Localizer.LocalizeByID("crud.cmd.flag.output.description"))
```
Comment (wtrocki) [inline pkg/template/crud/create.go:33]: how is this replaced?

Comment (namit-chandwani) [reply pkg/template/crud/create.go:33]: Sorry, didn't get you. Can you please elaborate? :)

Comment (wtrocki) [reply pkg/template/crud/create.go:33]: crud.cmd.flag how is that replaced to commandName.cmd.flag etc.

Comment (namit-chandwani) [reply pkg/template/crud/create.go:33]: No, we're not replacing that with the command name.

Through PR https://github.com/aerogear/charmil/pull/172, I have included a locale file as a template ([Link to the locale template file](https://github.com/aerogear/charmil/blob/main/pkg/template/crud/crud.en.yaml)).

This locale file even contains entries for every flag that is present in the CRUD commands (including [crud.cmd.flag.output.description](https://github.com/aerogear/charmil/blob/90e7e3fd8adbac493a1f3368cebee1bda1264f5e/pkg/template/crud/crud.en.yaml#L73)).

Whenever the `charmil crud` command is run, those 5 CRUD files will be generated in the path specified by user whereas this locale file will be generated in the `locale/en` directory of the CLI.

Comment (wtrocki) [reply pkg/template/crud/create.go:33]: That needs to be fixed

**Pull request merged**